Whatsapp_Automation/whatsappAutomation.py

```diff
@@ -0,0 +1,36 @@
+# Selenium is required for automation
+# sleep is required to have some time for scanning
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from time import sleep
+
+
+def whatsapp(to, message):
+    person = [to]
+    string = message
+    chrome_driver_binary = "C:\\Program Files\\Google\\Chrome\\Application\\chromedriver.exe"
+    # Selenium chromedriver path
+    driver = webdriver.Chrome(chrome_driver_binary)
+    driver.get("https://web.whatsapp.com/")
+    sleep(15)
+    # This will find the person we want to send the message to in the list
+    for name in person:
+        user = driver.find_element_by_xpath("//span[@title='{}']".format(name))
+        user.click()
+        text_box = driver.find_element_by_xpath(
+            '//*[@id="main"]/footer/div[1]/div[2]/div/div[2]')
+        text_box.send_keys(string)
+        sendbutton = driver.find_elements_by_xpath(
+            '//*[@id="main"]/footer/div[1]/div[3]/button')[0]
+        sendbutton.click()
```

Comment:
Print a response here on successful message sent action. You could add try catch here!!